source/robot_model/include/robot_model/Exceptions/FrameNotFoundException.hpp

```diff
@@ -1,13 +1,11 @@
 #pragma once
 
 #include <exception>
-#include <iostream>
 
-namespace RobotModel {
-namespace Exceptions {
-class FrameNotFoundException : public std::runtime_error {
+namespace RobotModel::Exceptions {
+class FrameNotFoundException : public std::invalid_argument {
```

Comment:
I was not aware this exception existed in std. Good catch. It should be used at some other places as well.